image/templates/helm/stackrox-secured-cluster/internal/defaults/10-env.yaml.htpl

```diff
@@ -1,4 +1,9 @@
 # This file applies default environment configuration, based on available API server resources.
+{{- if kindIs "invalid" ._rox.env.gRPCEnforceALPN }}
+env:
```

Comment:
`env` may be defined twice (here and in istio section). Is that intentional? I believe this renders invalid yaml?

Reply:
I thought it would end up in two yaml document patches applying on top of each other. I'll merge the logic here.